pkg/networkmanager/interfaces.js

```diff
@@ -125,29 +125,6 @@ export function connection_settings(c) {
  * TODO - document the details of the data structure.
  */
 
-/* HACK
```

Comment:
I think it's not good to remove the comment but not remove the actual HACK.  The code still is written with the assumption that NetworkManager does property notifications "wrong".  I think we should update the comment to state that it is out of date and we could now rewrite the whole Networking page with regular DBusProxies.

Reply:
Alright, updated!